documentation/general/design/dash-holistic-view.md

```diff
@@ -0,0 +1,618 @@
+  that capitalizes on the use of **programmable networking devices** including
+  - **SmartNICs**. A smart network interface card (SmartNIC), is a programmable
+    accelerator enabling flexible and efficient data center networking, security and storage
+    It offloads from server CPU a set of tasks such as
+    how to process and route packets of data.
+  - **SmartSwitches**. A smart switch is a network switch with programmable DPUs/IPUs/EPUs, etc...) SmartNIC hardware integrated into the switch design, with a selected number of options for management. 
+  - **SmartAppliances**. An appliance is a multi-tenant network appliance. It
+    has multiple data processing units (DPUs) supporting virtual ports.
+- Optimize **stateful L4** performance and connection scale by 10x or even 100x,
+  when compared to implementations that make extensive use of a generic software
+  stack approach (compromising performance for flexibility). As host networking
+  in the cloud is performed at L4, the resulting performance improvements should
+  be truly significant.
+- As a starting point, Microsoft Azure will integrate and deploy DASH solutions to
+  ensure that scale, monitoring, reliability, availability and constant
+  innovation are proven and hardened. Other enterprise and cloud providers may
+  deploy DASH as well.  It should be noted that innovations for **in-service
+  software upgrades** (ISSU) and **high availability** (HA) are key tenets of
+  the DASH charter.
+  
+## Compliance and requirements
+
+> [!NOTE] TBD - Some intro here from SDN packets transform document.
+
+The following are the compliance and requirements details applicable to the
+supported scenario:
+
+- [VNET to VNET
+  service](https://github.com/Azure/DASH/blob/main/documentation/vnet2vnet-service/requirements/README.md)
+- [VNET peering
+  service](https://github.com/Azure/DASH/blob/main/documentation/vnet-peering-service/requirements/README.md)
+- [Service tunnel & Private link
+  service](https://github.com/Azure/DASH/blob/main/documentation/stpl-service/requirements/README.md)
+- [Load balancer
+  service](https://github.com/Azure/DASH/blob/main/documentation/load-bal-service/requirements/README.md)
+- [Encryption gateway
+  service](https://github.com/Azure/DASH/blob/main/documentation/encrypt-gw-service/requirements/README.md)
+- [Express route gateway
+  service](https://github.com/Azure/DASH/blob/main/documentation/express-route-service/requirements/README.md)
+
+## Scenarios
+
+DASH extends SONiC APIs and a related comprehensive set of object models
+described in both *documentation* and a *behavioral model* to describe
+networking services for the cloud. The 7 initial services (in draft) describe
+this in more detail. We will also work with cloud and enterprise providers to
+further extend DASH to meet their specific needs.
+
+1. [VNET-to-VNET](https://github.com/Azure/DASH/tree/main/documentation/vnet2vnet-service).
```

Comment:
I think this should be described, it seems to be the basis for a  lot of general DASH services.

Reply:
@chrispsommers I added some wording. Please, verify its correctness. If you have a better explanation, please let me know. Thanks

Reply:
LGTM but I'd tighten the wording.